tests/sqlparser_clickhouse.rs

```diff
@@ -1422,6 +1422,38 @@ fn parse_create_table_on_commit_and_as_query() {
     }
 }
 
+#[test]
+fn parse_update_data() {
+    let sql = "ALTER TABLE t UPDATE col1 = 1, col2 = col3 + col4 WHERE cod4 = 1";
```

Comment:
Can we add a test case with `IN PARTITION`?

Reply:
Added.